### ISSUE TITLE
Add option to skip 403 retries, for content that is legitimately not available

### ISSUE THREAD

**Issue by fake-name** [dteviot/WebToEpub#1491]
Basically, I have a number of stories where LARGE swathes content has gone inaccessible. For example, there are a number of series where the author has (rather foolishly) used direct links to their patreon-hosted images in a publicly available story.

Downloading these is extremely annoying, because you get a "you might need to log in to a site" prompt for EVERY ONE of the broken links. You can wind up literally having to click "Yes, I know" more then a hundred times for a single series.

This is extremely annoying. 

It seems to me, given there is already a facility for deciding whether to ask the user (`wrapOptions.retry.promptUser`), that an option that just globally forces that to `false` would be very nice from a user standpoint.

**Comment by fake-name** [comment]
For the moment, I've kind of "redneck patched" the issue by just removing all `alert()` calls in `getAutomaticRetryBehaviourForStatus()`, and changing all `promptUser` values to `false`, but that's not a general fix. Ideally this would be something exposed through the UI, but I'm not smart enough to make that set of changes myself.

**Comment by gamebeaker** [comment]
@fake-name do you have an example story with problematic content?

**Comment by fake-name** [comment]
Try https://www.scribblehub.com/series/945582/the-heavenly-martial-empress-returns-an-op-xianxia-returnee-litrpg/

**Comment by gamebeaker** [comment]
https://www.scribblehub.com/series/945582/the-heavenly-martial-empress-returns-an-op-xianxia-returnee-litrpg/
Chapter 36 is problematic.

**Comment by gamebeaker** [comment]
The current flowchart for HTTP 403:
![Neues draw io Diagramm-93867 drawio](https://github.com/user-attachments/assets/cd5cf352-5713-4900-8d52-7be9e6930d68)

**Comment by gamebeaker** [comment]
@fake-name I changed how 403 gets handled is the new solution ok?
Test versions for Firefox and Chrome have been uploaded to https://github.com/dteviot/WebToEpub/releases/tag/developer-build. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* https://www.scribblehub.com/series/945582/the-heavenly-martial-empress-returns-an-op-xianxia-returnee-litrpg/
